src/backend/state.rs

```diff
@@ -461,10 +461,10 @@ impl AppState {
             return state;
         };
 
-        if let Some(private_key) = &config.wallet_private_key {
-            let wallet_state = &app_state.loaded_wallet;
-            add_wallet_by_private_key(&wallet_state, private_key).await;
-        }
+        // if let Some(private_key) = &config.wallet_private_key {
```

Comment:
it doesn't seem to be "required", if it's None, then it wouldn't execute it, no?

Reply:
quite possible, I suppose we better keep this functionality, but can handle empty string case better instead